mergify_engine/utils.py

```diff
@@ -112,6 +124,32 @@ def get_installation_id(integration, owner):
             return install["id"]
 
 
+def get_subscription(r, installation_id):
+    sub = r.hgetall("subscription-cache-%d" % installation_id)
+    if sub is None:
+        LOG.info("Subscription for %s not cached, retrieving it..." %
+                 installation_id)
+        resp = requests.get("https://mergify.io/engine/token/%s" %
```

Comment:
URL seems wrong, should be `/engine/installation/%d`

Reply:
Thx, I didn't notice it change